google-cloud-spanner/src/main/java/com/google/cloud/spanner/InstanceInfo.java

```diff
@@ -68,8 +77,22 @@ static FieldMask toFieldMask(InstanceField... fields) {
 
     public abstract Builder setDisplayName(String displayName);
 
+    /**
+     * Sets the number of nodes for the instance. Only one of processing units or node count must be
+     * set when creating a new instance.
+     */
     public abstract Builder setNodeCount(int nodeCount);
 
+    /**
+     * Sets the number of processing units for the instance. Only one of processing units or node
+     * count must be set when creating a new instance. Processing units must be between 1 and 999
+     * (inclusive) when creating a new instance with node count = 0. Processing units from 1000 and
+     * up must always be a multiple of 1000 (i.e. equal to an integer number of nodes).
```

Comment:
i.e. --> "That is" per google tech writing style guide

Reply:
Changed the wording to "that is" instead of "i.e."